bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,502 @@
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="sePreHeatingOn" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Pre Heating State</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seExhaustFanOff" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Stop Exhaust Fan</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seFirePlaceBoosterSwitch" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Fire Place Booster State</label>
+		<description>Indicates if fire booster switch is turned on</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seIncomingCurrent" advanced="true">
+		<item-type>Number</item-type>
+		<label>Incoming Current</label>
+		<description>Incoming current message in mA if remote control is activated</description>
+		<state min="0" max="255" step="1" pattern="%d mA" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seLastErrorNumber" advanced="true">
+		<item-type>Number</item-type>
+		<label>Last Error Number</label>
+		<description>Number of the alarm type that occurred last</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+
+	<channel-type id="seInstalledCo2Sensors" advanced="true">
+		<item-type>String</item-type>
+		<label>Installed Co2 Sensors</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingOnCounter" advanced="true">
+		<item-type>Number</item-type>
+		<label>Post Heating ON Counter</label>
+		<description>Time how long post heating will be ON. In seconds</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingOffCounter" advanced="true">
+		<item-type>Number</item-type>
+		<label>Post Heating OFF Counter</label>
+		<description>Time how long post heating will be OFF. In seconds</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingTargetValue" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Post heating target value</label>
+		<description>Target value for post heating</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2Alarm" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Co2 Alarm</label>
+		<description>Indicates if Co2 levels are too high</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHrcFreezingAlarm" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Heat recovery cell freezing alarm</label>
+		<description>Indicates if heat recovery cell is about to freeze</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seWaterRadiatorFreezingAlarm" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Water radiator freezing alarm</label>
```

Comment:
actually, this might be the last one

Reply:
Found few others but now they are sorted.